src/legacy/core_plugins/kibana/public/management/sections/settings/index.js

```diff
@@ -82,13 +83,18 @@ uiRoutes.when('/management/kibana/settings/:setting?', {
   },
 });
 
-uiModules.get('apps/management').directive('kbnManagementAdvanced', function(config, $route) {
+uiModules.get('apps/management').directive('kbnManagementAdvanced', function($route) {
   return {
     restrict: 'E',
     link: function($scope) {
-      config.watchAll(() => {
```

Comment:
Can you delete `watchAll` from `src/legacy/ui/public/config/config.js` as its no longer used?

Reply:
I'd prefer to decommission the old api in one go.